ui/client/common/SystemUtils.js

```diff
@@ -38,6 +40,14 @@ class SystemUtils {
       return config;
     });
   }
+
+  getUserId() {
+    if (!localStorage.getItem(userId)) {
```

Comment:
Better way will be `this.getItem(userId) !== null`

Reply:
I've changed it to _ .isEmpty... - if the user key will be "" it also not acceptable